data/multiplayer/scenarios/4p_King_of_the_Hill.cfg

```
@@ -6,7 +6,7 @@
     id=multiplayer_King_of_the_Hill
     name= _ "4p — King of the Hill"
     map_file=multiplayer/maps/4p_King_of_the_Hill.map
-    description= _ "Controlling the area around the central keep is very lucrative in this 4 player FFA map. Works fine 2 vs. 2 as well. There are 36 villages." + _ " Recommended setting of 2 gold per village and 150 starting gold."
+    description= _ "The King is dead. Without an heir. his quarrelsome lords now vie for the hilltop throne and all the power and wealth it commands. Are you mighty enough to take this seat of power before your rivals do? Can you hold out against their many assassins? Is your arse truly great enough to fill the throne of a king? This 4 player FFA map works fine 2 vs. 2 as well. There are 38 villages and bonus income for a leader on the center keep." + _ " Recommended setting of 2 gold per village and 150 starting gold."
```

Basic fix:

```suggestion
    description= _ "The King is dead. Without an heir, his quarrelsome lords now vie for the hilltop throne and all the power and wealth it commands. Are you mighty enough to take this seat of power before your rivals do? Can you hold out against their many assassins? Is your arse truly great enough to fill the throne of a king? This 4 player FFA map works fine 2 vs. 2 as well. There are 38 villages and bonus income for the leader on the center keep." + _ " Recommended setting of 2 gold per village and 150 starting gold."
```

I also don't think the "arse" reference is appropriate (not to mention it's a Britishism).

---

Suggestion applied, thanks.